src/components/IncreamentDecreament.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import InputSpinner from "react-native-input-spinner";
+import {StyleSheet, Text, View} from "react-native";
+import {colors} from "../constants";
+
+const IncrementDecrement = ({title,value,max,setValue}: any) => {
+    return (<>
+            <View style={styles.textView}><Text>{title}</Text></View>
+            <View style={styles.container}>
+                <InputSpinner
```

Comment:
Change to `Spinner` (or another alias name as which you imported this component)